### ISSUE TITLE
Checkbox label text is not announced when hovering over label or input, with NVDA mouse tracking enabled

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/reported-bugs#70]
**Upstream bug:** https://github.com/nvaccess/nvda/issues/13879
**Date:** 2022-07-08
**Reported by:** Vanita Barrett-Smith @vanitabarrett
**Related to:** https://github.com/alphagov/govuk-frontend/issues/2278

---

### Overview

Checkboxes only read out the label when you hover over the checkbox, not when you hover over the label (NVDA version 2018.1)

Checkboxes not announced at all when hovering over label or input (NVDA version 2019.1 and above)

### Expected result

NVDA should announce the checkbox label

Note: in version 2018.1 of NVDA, the checkbox label is announced when hovering over the checkbox input, but this doesn't happen in versions 2019.1 and above.

### Actual result

NVDA does not announce the checkbox label when hovering over the label or the checkbox input, even with mouse tracking enabled
